ui_testing/testcases/base_test.py

```diff
@@ -40,27 +21,7 @@ def setUp(self):
         self.login_page = Login()
         self.test_plan = TstPlan()
```

Comment:
move it to the related method. i,e move it to `get_active_article_with_tst_plan` and  `get_multiple_active_article_with_tst_plan` do the same for the rest objects. Make sure that the related modules have it in its setup

Reply:
done